pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -98,6 +98,9 @@ type ScrapeConfigSpec struct {
 	// HonorLabels chooses the metric's labels on collisions with target labels.
 	// +optional
 	HonorLabels *bool `json:"honorLabels,omitempty"`
+	// Optional HTTP URL parameters
+	// +optional
+	Params *map[string][]string `json:"params,omitempty"`
```

Comment:
we don't need a pointer for optional maps and slices.

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required

```suggestion
	Params map[string][]string `json:"params,omitempty"`
```